Dockerfile

```diff
@@ -0,0 +1,37 @@
+# sources:
+# - https://github.com/jupyter/docker-stacks
+# - https://github.com/TomographicImaging/CIL#installation-of-cil
+# consumers:
+# - https://github.com/stfc/cloud-docker-images
+# TODO: use `ubuntu-22.04` after `python=3.11` is supported, vis.
+# https://github.com/TomographicImaging/CIL/issues/1490
+FROM jupyter/tensorflow-notebook:ubuntu-20.04
+LABEL org.opencontainers.image.source=https://github.com/TomographicImaging/CIL
+LABEL org.opencontainers.image.description="Core Imaging Library"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
```

Comment:
If we distribute with ASTRA the license is GPL-v3 as ASTRA.

I suggest that our default image is distributed with TIGRE only, and it is Apache-2.0. Then we add an extra `FROM` and modify the image/conda environment installing ASTRA and we distribute it again as GPL-v3.

Reply:
actually TIGRE is BSD-3-Clause :)